### PR TITLE
add swapfile to Vagrant

### LINKED ISSUE #983: Running all tests under Vagrant erros with out of memory
## Environment details:

* vagrant on Mac OS

## Steps to reproduce the issue:

1.   Clone repo and run openwhisk/tools/vagrant/hello
2.   Vagrant ssh in and run ./gradlew tests:test

## Provide the expected results and outputs:

All tests to run and pass

## Provide the actual results and outputs:

Out of memory errors on the JVM

## Additional information you deem important:

After looking into it, this is due to zero swap space configured for the Ubuntu VM.
The solution is to configure swap space in  the Vagrant file

### PR DESCRIPTION
**Pull request by csantanapr**
Fixes #983

### CODE REVIEW

**Comment by rabbah** [comment]
consolidate ansible/.gitignore with the one at top level?

File: tools/vagrant/Vagrantfile [hunk 1]
```diff
@@ -52,6 +52,15 @@ $script_common = <<SCRIPT
   sudo apt-get update
   sudo apt-get install git -y
   sudo apt-get install htop
+  
+  #Configure swapfile
```
Comment (rabbah) [inline tools/vagrant/Vagrantfile]: -> # Configure ...

Comment (csantanapr) [inline tools/vagrant/Vagrantfile]: Reviewing comments? Being very picky or too much free time on your hand my friend? 😺

Comment (rabbah) [inline tools/vagrant/Vagrantfile]: Time is all I have 😉

**Comment by csantanapr** [comment]
@bjustin-ibm addressed @rabbah review comments, I think is ready for merge
Thanks @rabbah for the quick review. 👍

**Comment by bjustin-ibm** [comment]
@csantanapr I restarted the Travis build after a failure that I suspect is unrelated to your change.

**Comment by csantanapr** [comment]
>@csantanapr I restarted the Travis build after a failure that I suspect is unrelated to your change.

@bjustin-ibm Trying to be funny? 😺 
Vagrant is not used in travis.

Maybe the scancode for the .gitignore or the Vagrant file?
Will wait and see what happens with the Travis build then.

**Comment by bjustin-ibm** [comment]
Rebased and merged as 8c265dccb600ef84d6a83225073dcb148580a6d9

**Pull request closed without merging**